src/compensations/Compensations.tsx

```diff
@@ -2,30 +2,35 @@
 import styles from "./compensations.module.css";
 import Text from "src/components/text/Text";
 import { CompensationsPage } from "studio/lib/payloads/compensations";
-import { RichText } from "src/components/richText/RichText";
 import SalaryCalculator, {
   Degree,
 } from "./components/salaryCalculator/SalaryCalculator";
 import { useState } from "react";
 import { calculatePension, calculateSalary } from "./utils/calculateSalary";
+import { CompanyLocation } from "studio/lib/payloads/companyDetails";
+import {
+  IOption,
+  RadioButtonGroup,
+} from "src/components/forms/radioButtonGroup/RadioButtonGroup";
 
 interface CompensationsProps {
   compensations: CompensationsPage;
+  locations: CompanyLocation;
```

Comment:
```suggestion
  locations: CompanyLocation[];
```

or

```suggestion
  location: CompanyLocation;
```

Reply:
It should be a list, so first suggestion :+1: